src/components/Box/Box.stories.tsx

```diff
@@ -0,0 +1,36 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import { Box as BoxComponent } from './Box';
+
+type Props = JSX.IntrinsicElements['div'];
+
+export default {
+    title: 'Layout/Box',
+    component: BoxComponent,
+    tags: ['autodocs'],
+    argTypes: {
+        className: {
+            type: 'string',
+        },
+        'data-test-id': {
+            type: 'string',
+        },
+        'aria-label': {
+            type: 'string',
+        },
+    },
+    args: {
+        className: 'tw-font-bold tw-flex',
```

Comment:
Why do we allow this, Fondue is a design system not just an UI library 🤔

Reply:
On our feedback sessions we heard a lot from devs that exposing the `className` here were a must for them, also designers agreed that this is necessary since it is part of our Design System Principles.